src/util/rocksdb/engine_metrics.rs

```diff
@@ -955,6 +956,20 @@ pub fn flush_engine_properties(engine: &DB, name: &str) {
                     .set(v);
             }
         }
+
+        // Num files at levels
+        let opts = engine.get_options_cf(handle);
+        for level in 0..opts.get_num_levels() {
+            let level_str = level.to_string();
```

Comment:
You can use `format!` here, have a look at `rocksdb::get_engine_compression_ratio_at_level`.